packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useModifiersStore.js

```diff
@@ -60,6 +60,7 @@ export function useModifiersStore() {
     } else {
       Editor.addMark(editor, name, true);
     }
+    ReactEditor.focus(editor);
```

Comment:
Why does it loose focus after a baseHandleToggle? I also have no idea what a baseHandleToggle is.

Reply:
I added the JSDoc and handled this feedback on https://github.com/strapi/strapi/pull/18193